File: Sprint 12/Modules 2&3/Astronomy-Objc/Astronomy-Objc/Model/SolDescription.m
//
//  SolDescription.m
//  Astronomy-Objc
//
//  Created by Nikita Thomas on 1/28/19.
//  Copyright © 2019 Nikita Thomas. All rights reserved.
//

#import "SolDescription.h"

@implementation SolDescription

- (instancetype)initWithSolNumber:(NSInteger)sol totalPhotos:(NSInteger)totalPhotos cameras:(NSArray<NSString *> *)cameras

{
    self = [super init];
    if (self) {
        _sol = sol;
        _totalPhotos = totalPhotos;
        _cameras = [cameras copy];
    }
    return self;
}




@end

File: Sprint 12/Modules 2&3/Objc Mars Rover/Objc Mars Rover/Models/SolDescription.swift
//
//  SolDescription.swift
//  Objc Mars Rover
//
//  Created by Sergey Osipyan on 3/4/19.
//  Copyright © 2019 Sergey Osipyan. All rights reserved.
//

import UIKit

class SolDescription: NSObject, Codable {
    
        let sol: Int
        let totalPhotos: Int
        let cameras: [String]
    
    init(sol: Int, totalPhotos: Int, cameras: [String]) {
        self.sol = sol
        self.totalPhotos = totalPhotos
        self.cameras = cameras
    }
}
